tests/test_fetch.py

```diff
@@ -0,0 +1,399 @@
+import pytest
+from typing import List
+from operator import itemgetter
+import itertools
+import numpy as np
+import decimal
+import pandas
+import warnings
+from . import schema
+import datajoint as dj
+import os
+import logging
+import io
+
+
+@pytest.fixture
+def lang():
+    yield schema.Language()
```

Comment:
What is the purpose of accessing `schema.Language()` through a fixture vs directly? I tested directly accessing the table and the suite passed.

We also stick with the `setup_class` method for the same scenario in [test_autopopulate.py](https://github.com/datajoint/datajoint-python/blob/dev-tests/tests/test_autopopulate.py#L13-L20) and [test_relation_u.py](https://github.com/datajoint/datajoint-python/blob/dev-tests/tests/test_relation_u.py#L13-L23). 

I think it's important to maintain a consistent pattern for recurring scenarios. That way we establish a standard for future cases.

Reply:
The purpose is consistency. We want to control the state that each test receives, isolating at the test level as much as possible. In some cases, like the `languages` fixture below, this is necessary (https://github.com/datajoint/datajoint-python/pull/1129/files#r1424702143) because the fixture actually changes the table instance.